<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QWidget" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>首页</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*
{
    font-family: &quot;Microsoft YaHei&quot;;
}

QLabel#lbPerson
{
	image: url(:/image/person.png);
}
QLabel#lbHeart
{
	image: url(:/image/heart.png);
}
QLabel#lbScore
{
	background-color: #EFF6FF;
	color: #2563EB;
	font:16px;
	border-radius:12px;
}

QScrollArea
{
	border:none;
}

QFrame#frameHome
{
	background-color: rgb(244, 245, 246);
	border-top-left-radius:12px;
	border-top-right-radius:12px;
}

QFrame#frameHomeTitle
{
	background-color: rgb(255, 255, 255);
	border-top-left-radius:12px;
	border-top-right-radius:12px;
}

QProgressBar 
{
	background-color: #f0f0f0;
	border-radius: 5px;
	text-align: center;
}

QProgressBar::chunk 
{
	border-radius: 5px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
         stop:0 #4a90e2, 
         stop:1 #57d9a3);
}

QGroupBox
{
	background-color: rgb(255, 255, 255);
	border-radius:12px;
}

QGroupBox#groupQuickTest
{
background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                       stop:0 #4388F4, /* 渐变起始色（蓝） */
                                       stop:1 #2ecc71);/* 渐变结束色（绿） */
}

QGroupBox#groupQuickTest QLabel
{
	color: rgb(255, 255, 255);
}

QPushButton#btnStartTest
{
	background-color: rgb(255, 255, 255);
	color: #2563EB;
	border-radius:20px;
	font:14px;
}


QScrollBar:vertical 
{
	border:0px solid red;
	background-color:#FAFAFA;
	width:6px;
	margin:0 0 0 0px;
}

QScrollBar::add-line:vertical 
{
 border:none;
 background:transparent; 
 height:0px;
}
 /* 顶部控制块 */ 
QScrollBar::sub-line:vertical { 
 border:none; 
 background:transparent; 
 height:0px; 
 subcontrol-position:top;
 subcontrol-origin:margin;
 
}
/* 滑块设置 */
QScrollBar::handle:vertical {
	background: #DCDFE6;
	min-height:200px;
	border-radius: 3px;
}
 /* 滑块上边区域 */ 
QScrollBar::add-page:vertical {
 background:transparent;
} 
/* 滑块下边区域 */
 QScrollBar::sub-page:vertical {
 background:transparent; 
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameHome">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frameHomeTitle">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <item>
          <widget class="QLabel" name="lbHeart">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelLogo">
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>心屿</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>309</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>400</width>
           <height>665</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>15</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupUserStatus">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>130</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>14</number>
               </property>
               <property name="topMargin">
                <number>15</number>
               </property>
               <property name="rightMargin">
                <number>14</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <property name="spacing">
                  <number>10</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="lbPerson">
                   <property name="minimumSize">
                    <size>
                     <width>50</width>
                     <height>50</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>50</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <item>
                    <widget class="QLabel" name="labelUserName">
                     <property name="font">
                      <font>
                       <family>Microsoft YaHei</family>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>心灵旅人</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="font">
                      <font>
                       <family>Microsoft YaHei</family>
                       <pointsize>11</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>今日情绪指数</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QLabel" name="lbScore">
                   <property name="minimumSize">
                    <size>
                     <width>50</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>76</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>平静</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>积极</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QProgressBar" name="progressMood">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>10</height>
                  </size>
                 </property>
                 <property name="value">
                  <number>76</number>
                 </property>
                 <property name="textVisible">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupQuickTest">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>150</height>
               </size>
              </property>
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>今日心里测评</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelQuickTestDesc">
                 <property name="text">
                  <string>只需几分钟，了解你的当下情绪。</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QPushButton" name="btnStartTest">
                   <property name="minimumSize">
                    <size>
                     <width>126</width>
                     <height>45</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>126</width>
                     <height>45</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>开始今日测评</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupTrend">
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>情绪趋势</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>近7天</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="labelTrendPlaceholder">
                 <property name="text">
                  <string>这里预留绘制情绪趋势图的位置。</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupSuggestion">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>100</height>
               </size>
              </property>
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>每日心理小tips</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="labelSuggestion">
                 <property name="text">
                  <string>根据你的情绪数据，这里会给出个性化的小建议。</string>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
